scss/components/_soe_dm_collection.scss

```diff
@@ -0,0 +1,205 @@
+@charset "UTF-8";
+
+//
+// DIGITAL MAGAZINE CCOLLECTIONS
+// LANDING PAGE
+//
+
+// COLLECTIONS LANDING PAGE
+
+.page-magazine-collections,
+.node-type-stanford-collection {
+  #content-body-bottom {
+    margin-top: 60px;
+  }
+
+  #mc_embed_signup_scroll {
+    padding-top: 100px;
+    padding-bottom: 100px;
+  }
+
+  .main {
+    padding: 0;
+  }
+
+  .view-display-id-collections_page {
+    margin: 0 10%;
+    table {
+      tr {
+        td {
+          border: none;
+        }
+      }
+    }
+
+    .view-header {
+      h1 {
+        font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        color: #686868;
+        font-size: 1.3em;
+        text-align: center;
+        margin-bottom: 0;
+      }
+
+      h2 {
+        font-size: 2.8em;
+        text-align: center;
+        margin: 0 0 60px;
+        width: 100%;
+      }
+    }
+
+    .mag-collection-container {
+      background-color: #fff;
+      float: left;
+      width: 100%;
+      -webkit-box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2);
+      -moz-box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2);
+      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2);
+
+      .mag-collection-img {
+        float: left;
+      }
+
+      .mag-collection-content-container {
+        float: left;
+        padding: 25px 10px 0 40px;
+        width: 50%;
+
+        .mag-collection-title {
+          font-size: 1.3em;
+          line-height: 1.3em;
+          font-weight: 600;
+          margin-bottom: 25px;
+          margin-top: 10px;
+
+          a {
+            color: #B1040E;
+          }
+        }
+
+        .mag-collection-superhead {
+          color: #686868;
+          font-size: 0.9em;
+          font-weight: 100;
+        }
+
+        .mag-collection-lead {
+          font-size: 0.8em;
+          font-weight: 400;
+          line-height: 1.2em;
+          margin: 20px 0;
+        }
+      }
+    }
+  }
+}
+
+//
+// A DIGITAL MAGAZINE COLLECTION PAGE
+//
+
+// A COLLECTION PAGE
+
+.node-type-stanford-collection {
+  #content-head {
+    display: block;
+    #page-title {
+      display: none;
```

Comment:
I am not a fan of hiding titles with display: none. Robots and screen readers can still pick these up sometimes. Also, it causes problems for the next developer with discoverability. When they go to change the page title with a hook_alter or something else it wont visibly take effect causing troubleshooting time and overhead that could be avoided. I would like you to find a different approach to the page title. 

Why do you need to provide your own in the view?